util-liquibase-installer/pom.xml

```diff
@@ -46,11 +46,22 @@
         <dependency>
             <groupId>org.springframework</groupId>
             <artifactId>spring-core</artifactId>
+            <exclusions>
+                <exclusion>
+                    <artifactId>commons-logging</artifactId>
+                    <groupId>commons-logging</groupId>
+                </exclusion>
+            </exclusions>
```

Comment:
Remove from here since this should be covered at the dependency management level.

Reply:
removed